src/app/pages/cases/details/DetailsPage.tsx

```diff
@@ -54,19 +53,17 @@ const DetailsPage: React.FC<RouteComponentProps<Props>> = ({ id: idString }) =>
       <RowWithColumn>
         <PageHeading />
       </RowWithColumn>
-      <Row>
-        <Column spanLarge={50}>
-          <CaseDetails caseId={ id! } />
-        </Column>
-      </Row>
+      <RowWithColumn>
```

Comment:
The definitionlist will be 100% wide now, is that on purpose?

Reply:
Yes, didn't want to line break the longer UUID. Do agree it's not looking that great on wider screens, though.